styles/globals.css

```
@@ -173,23 +173,73 @@
+     These are VARIABLE fonts covering weight 300-900 in one file per subset,
+     so `font-weight: 300 900` lets every weight the design uses resolve from
+     a single download instead of one file per weight.
+
+     Split by subset with Google's own unicode-range values: the browser only
+     fetches the Arabic file when Arabic codepoints are actually painted, and
+     only fetches the Latin file when Latin appears (brand names like Signal,
+     URLs, numerals inside Arabic copy). Combined with the [lang="ar"] scoping
+     further down, an English or Spanish reader downloads none of this. */
+  @font-face {
+    font-display: swap;
+    font-family: 'Rubik';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Rubik" (font-family-name-quotes)

(font-family-name-quotes)